proto/viam/app/data/v1/data.proto

```diff
@@ -145,6 +145,7 @@ message TabularDataByFilterResponse {
   repeated TabularData data = 2;
   int64 count = 3;
   string last = 4;
+  int64 total_size = 5;
```

Comment:
Two super nitty things: 
- This is the size in bytes, right? If so, I think it might make sense to append _bytes to the end to make that super explicit/obvious
- I think since this should always be a non-negative number, we can use a uint here for some added safety

Reply:
Sounds good, I can rename this. 

Also happy to change this to a uint. FWIW I chose int64 because I saw other positive numbers (such as count above) using int64, and the FileSizeBytes field in the TabularCapture is also an int64, which means we don't have to to cast those to uints when calculating the total size.

Reply:
Yeah that makes sense, we're definitely inconsistent here - we use uint below for all the binary + delete rpcs. I think that's definitely fair to leave this as an int for consistency within these Tabular rpcs, and then we can just migrate them all over together at some point in the future

Reply:
Thanks for pointing that out, I ended up changing the type to uint64. I'll file a ticket to change the type of the `count` field and the fields on the TabularCapture and link it here.

Reply:
Filed [DATA-1063](https://viam.atlassian.net/browse/DATA-1063).